Commands/notify.py

```diff
@@ -160,9 +160,10 @@ async def save_config(self, ctx):
         await save_configs(ctx)
         await ctx.send("Configurations saved")
 
-    @commands.command(name="add_list", brief="**admin-only** \n\u2003 Add a new list",usage='<list name>', help="*admin-only* \u2003 Add a new list. \n You will be asked what emoji to use for this list. React to the question of the bot with an emoji that is not yet used for another list. \n\n <list name> \u2003 The name of the list to add.")
+    @commands.command(name="add_list", brief="**admin-only** \n\u2003 Add a new list", usage='<list name>', help="*admin-only* \u2003 Add a new list. \n You will be asked what emoji to use for this list. React to the question of the bot with an emoji that is not yet used for another list. \n\n <list name> \u2003 The name of the list to add.")
     async def add_list(self, ctx, list_name):
-        if not ctx.message.author.guild_permissions.administrator:
+        guild_data = await get_guild_data(ctx.message.guild.id)
+        if not (ctx.message.author.guild_permissions.administrator or ctx.author.id in guild_data.bot_admins):
```

Comment:
in the if, both `ctx.message.author` and `ctx.author` are used, is there a reason for this? would expect it both to have the same, no?

Reply:
It appears that in the new version of discord.py `ctx.author` is a shortcut for `ctx.message.author` and `ctx.guild` is a shortcut for `ctx.message.guild`. I changed both

Reply:
Lovely, will use that in my branch as well then!